doc/whats_new/v0.24.rst

```diff
@@ -53,6 +53,14 @@ Changelog
   sparse matrix or dataframe at the start. :pr:`17546` by
   :user:`Lucy Liu <lucyleeow>`.
 
+:mod:`sklearn.cluster`
+.........................
+
+- |Fix| Fixed a bug in :class:`cluster.KMeans` and
```

Comment:
Since the inertia will change, we should add this change in the `Changed Model` section.